internal/cli/cmd/cluster/describe.go

```diff
@@ -225,44 +224,35 @@ func showEndpoints(c *appsv1alpha1.Cluster, svcList *corev1.ServiceList, out io.
 	tbl.Print()
 }
 
-func showDataProtection(backupPolicies []dpv1alpha1.BackupPolicy, backups []dpv1alpha1.Backup, out io.Writer) {
-	if len(backupPolicies) == 0 {
+func showDataProtection(backupPolicies []dpv1alpha1.BackupPolicy, backupSchedules []dpv1alpha1.BackupSchedule, out io.Writer) {
+	if len(backupPolicies) == 0 || len(backupSchedules) == 0 {
 		return
 	}
-	tbl := newTbl(out, "\nData Protection:", "AUTO-BACKUP", "BACKUP-SCHEDULE", "TYPE", "BACKUP-TTL", "LAST-SCHEDULE", "RECOVERABLE-TIME")
-	for _, policy := range backupPolicies {
-		if policy.Annotations[dptypes.DefaultBackupPolicyAnnotationKey] != "true" {
-			continue
+	tbl := newTbl(out, "\nData Protection:", "BACKUP-REPO", "AUTO-BACKUP", "BACKUP-SCHEDULE", "BACKUP-METHOD", "BACKUP-RETENTION")
+	for _, schedule := range backupSchedules {
+		backupRepo := printer.NoneString
+		backupMethod := printer.NoneString
+		backupSchedule := printer.NoneString
+		backupRetention := printer.NoneString
+		scheduleEnable := "Disabled"
+		for _, policy := range backupPolicies {
+			if policy.Name != schedule.Spec.BackupPolicyName {
+				continue
+			}
+			if policy.Spec.BackupRepoName != nil {
```

Comment:
If `policy.Spec.BackupRepoName` is nil, will use the default backupRepo, get the default backupRepo name and output it.